apps/condo/domains/ticket/utils/serverSchema/TicketChange.js

```diff
@@ -0,0 +1,163 @@
+                        }
+                    }
+                }
+            `,
+            variables: { id: existingItem.id },
+        })
+        if (updatedResult.errors) {
+            console.error('Error while fetching related items in relatedManyToManyResolvers of changeTrackable for a Ticket', updatedResult.errors)
+            return {}
+        }
+        if (updatedResult.data.ticket) {
+            updated = {
+                ids: _.map(updatedResult.data.ticket.watchers, 'id'),
+                displayNames: _.map(updatedResult.data.ticket.watchers, 'name'),
+            }
+        }
+
+        /*
+            Restore previous list of related items
+            This is a tricky part, because we need to restore previous list of related items,
+            that explicitly is not presented anywhere.
+            Following solutions was considered:
+            1. Get it from `beforeChange` hook of the Ticket, but where to store it?
+            2. Get previous TicketChange and get it from `watchersIdsTo` field.
+               This solution is not suitable for handling first change,
+               because there will be no `TicketChange` item yet.
+            3. Get current `watchers` list (after update) and make a replay,
+               based on Keystone `originalInput`, which stores "Nested mutation" operations
+               for this relationship. This variant is implemented.
+        */
+        let existing = _.cloneDeep(updated)
+        if (originalInput && originalInput.watchers) {
+            if (originalInput.watchers.disconnect) {
+                // Perform opposite operation
+                existing.ids = _.uniq([...existing.ids, ..._.map(originalInput.watchers.disconnect, 'id')])
+            }
+            if (originalInput.watchers.connect) {
+                // Perform opposite operation
+                existing.ids = _.difference(existing.ids, _.map(originalInput.watchers.connect, 'id'))
+            }
+        }
+        const usersResult = await context.executeGraphQL({
+            context: context.createContext({ skipAccessControl: true }),
+            query: `
+                query changeTrackable_findUsers($ids: [ID!]) {
+                    users: allUsers(where: { id_in: $ids }) {
+                        id
+                        name
+                    }
+                }
+            `,
+            variables: { ids: existing.ids },
+        })
+        if (usersResult.error) {
+            console.error('Error while fetching users in relatedManyToManyResolvers of changeTrackable for a Ticket', updatedResult.errors)
+            return {}
+        }
+        if (usersResult.data.users) {
+            existing.displayNames = _.map(usersResult.data.users, 'name')
+        }
+
+        return {
+            existing,
+            updated,
+        }
+    },
+}
+
+module.exports = {
+    createTicketChange,
+    displayNameResolvers,
+    relatedManyToManyResolvers,
```

Comment:
it's a too-common name. Use something like: `ticketChangeRelatedManyToManyResolvers `